utils/SwiftBuildSupport.py

```diff
@@ -12,12 +12,16 @@
 
 from __future__ import print_function
 
-import ConfigParser
 import os
 import pipes
 import subprocess
 import sys
 
+if sys.version_info < (3, 0):
+    import ConfigParser
+else:
+    import configparser as ConfigParser
```

Comment:
I mildly prefer checking the version, so that an ImportError in ConfigParser won’t cause Python 2 to swallow the exception and try to import configparser instead, but it’s not a strong preference.